### PR TITLE
Reorganize Overview section

### PR DESCRIPTION
**Pull request by kcgthb**
Reorganize and update Overview section to include ordering information

### CODE REVIEW

File: src/docs/overview/concepts.md [hunk 1]
```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
     For more information about data risk classifications, see the [Information
     Security Risk Classification page][url_risk].
 
-## What's a cluster?
 
-A computing cluster is a federation of multiple compute nodes (independent
-computers), most commonly linked together through a high-performance
-interconnect network.
+## Investing into Sherlock
```
Comment (vsoch) [inline src/docs/overview/concepts.md:62]: Maybe just "Investing in Sherlock?"

Comment (vsoch) [inline src/docs/overview/concepts.md:62]:
```suggestion
## Investing in Sherlock
```

File: src/docs/overview/concepts.md [hunk 2]
```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
...
 
-What makes it a "super-computer" is the ability for a program to address
-resources (such as memory, CPU cores) located in different compute nodes,
-through the high-performance interconnect network.
-
-![overview](img/cluster_overview.png)
+For users who need more than casual access to a shared computing environment,
+Sherlock also offers Faculty members the possibility to invest into additional,
```
Comment (vsoch) [inline src/docs/overview/concepts.md:65]:
```suggestion
Sherlock also offers Faculty members the possibility to invest in additional,
```

File: src/docs/overview/concepts.md [hunk 3]
```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
...
-requirements with available resources such as [CPUs][g_cpu], memory or
-computing accelerators such as [GPUs][g_gpu]. It will then execute the user
-defined tasks on the selected resources, and generate output files in one of
-the different [storage locations][url_storage] available on the cluster, for
-the user to review and analyze.
+Unlike traditional clusters, Sherlock is a collaborative system where the
+majority of nodes are purchased and shared by the cluster users, known as
+*owners*. Choosing from a standard set of server configurations supported by
+SRCC staff (known as the Sherlock *catalog*), PIs can purchase their own
+servers to add to the cluster.
```
Comment (vsoch) [inline src/docs/overview/concepts.md:72]:
```suggestion
majority of nodes are purchased and shared by the cluster users. When a user (typically a PI)
purchases one or more nodes, they become an *owner*. Owners can choose from a standard set of server configurations supported by SRCC staff (known as the Sherlock *catalog*) to add to the cluster.
```.

Comment (vsoch) [reply src/docs/overview/concepts.md:72]: I think the definition of "owner" isn't clear enough - the way it's currently written it is referencing generic cluster users.

File: src/docs/overview/concepts.md [hunk 4]
```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
...
-[g_cpu]:        /docs/overview/glossary#cpu
-[g_gpu]:        /docs/overview/glossary#gpu
-[g_login]:      /docs/overview/glossary#login-nodes
-[g_ssh]:        /docs/overview/glossary#ssh
-[g_jobs]:       /docs/overview/glossary#job
-[g_scheduler]:  /docs/overview/glossary#scheduler
-[url_storage]:  /docs/storage/
+This model also allows Sherlock owners to benefit from the scale of the cluster
+by giving them access to more compute nodes than their individual purchase,
+which gives them much greater flexibility than owning a standalone cluster.
```
Comment (vsoch) [inline src/docs/overview/concepts.md:76]: Could we add a line about why that is possible? E.g., "owners share a group of nodes that, when not in use by the primary purchaser of the machine, can be used by other owners."

File: src/docs/overview/concepts.md [hunk 5]
```diff
@@ -58,70 +58,44 @@ through a job scheduler using a [fair-share algorithm][url_fairshare].
...
-of magnitude increase in about 3 years.
-
-This model, often referred to as the **Condo model**, allows Sherlock owners to
-benefit from the scale of the cluster and give them access to more compute
-nodes than their individual purchase.  This provides owners with much greater
-flexibility than owning a standalone cluster.
+    The vast majority of Sherlock's compute nodes have been purchased by
+    individual PIs and groups, and PI purchases are the main driver behind the
+    rapid expansion of the cluster, which went from 120 nodes to more than
+    1,000 nodes in less than 3 years.
```
Comment (vsoch) [inline src/docs/overview/concepts.md:83]: damn

File: src/docs/overview/concepts.md [hunk 6]
```diff
@@ -200,15 +165,21 @@ generation of Sherlock.
...
+the system.
+
+
 !!! info "Maintenance schedule"
 
-     As often as possible, maintenances will take place on the **first Tuesday
-     of every month, from 08:00 to 12:00** (noon), and will be announced 2 weeks in
-     advance, through the usual communication channels.
+    As often as possible, maintenances will take place on the **first Tuesday
+    of every month, from 08:00 to 12:00** (noon), and will be announced 2 weeks
```
Comment (vsoch) [inline src/docs/overview/concepts.md:181]:
```suggestion
    of every month, from 08:00 to 12:00 Pacific time** (noon), and will be announced 2 weeks
```

File: src/docs/overview/concepts.md (hunk 3 above)
Comment (kcgthb) [reply src/docs/overview/concepts.md:72]: Agreed, thanks for the correction!

File: src/docs/overview/concepts.md (hunk 4 above)
Comment (kcgthb) [reply src/docs/overview/concepts.md]: It's detailed a little bit below with "The resource scheduler configuration works like this:", but I added a sentence to clarify.

**Pull request merged**